azure_functions_worker/loader.py

```diff
@@ -91,3 +93,32 @@ def load_function(name: str, directory: str, script_file: str,
             f'present in {rel_script_path}')
 
     return func
+
+
+@attach_message_to_exception(
+    expt_type=ImportError,
+    message=f'Troubleshooting Guide: {MODULE_NOT_FOUND_TS_URL}'
+)
+def index_function_app(directory: str) -> typing.List[Function]:
+    function_path = os.path.join(directory, SCRIPT_FILE_NAME)
+
+    if not os.path.exists(function_path):
+        raise IOError(f"{SCRIPT_FILE_NAME} not found in the function app")
```

Comment:
Should it be a FunctionsLoadError - for users to understand we could not load the functions? And in the future, add an aka.ms link to showcase the new programming model requirements and constraints?

(create a task for a reminder for aka.ms stuff)